ecosystem/python/sdk/aptos_sdk/client.py

```diff
@@ -489,7 +499,7 @@ def get_token_data(
             "0x3::token::TokenData",
             token_data_id,
         )
-        #<:!:read_token_data_table
+        # <:!:read_token_data_table
```

Comment:
Btw, I'm not sure if it needs a space or not for the tutorials, can you look at the ones in the typescript tutorials?

Reply:
These are added by my IDE formatter. I checked the generated doc. It looks fine.